benchmarks/models.py

```diff
@@ -176,6 +176,7 @@ class Model(models.Model):
     visual_degrees = models.IntegerField(null=True)  # null during model creation before querying objec
     public = models.BooleanField(default=False)
     competition = models.CharField(max_length=200, default=None, null=True)
+    domain = models.CharField(max_length=200, default="vision", null=True)
```

Comment:
```suggestion
    domain = models.CharField(max_length=200)
```
This should always be specified, and cannot be null